### ISSUE TITLE
APIs appear to be doing nothing

### ISSUE THREAD

**Issue by kevinguard** [matthewp/fs#12]
I am testing this on Chrome browser and basically APIs work without throwing any error however they seem to be doing nothing. For example, readFile or mkdir don't throw but the result is no-op. Is there anything I might be missing in terms of setup, configurations, etc.? Thanks!

**Comment by matthewp** [comment]
I don't think so. I haven't used this library in a while so I'm not sure, it's possible that something changed in Chrome that is breaking it. PRs welcome.

**Comment by kiko0217** [comment]
its not working anything

**Comment by dashuaidai** [comment]
> I am testing this on Chrome browser and basically APIs work without throwing any error however they seem to be doing nothing. For example, readFile or mkdir don't throw but the result is no-op. Is there anything I might be missing in terms of setup, configurations, etc.? Thanks!



> its not working anything

It update your indexDB, press f12 and check application/storage/indexDB to inspect your data.

**Comment by dashuaidai** [comment]
I think this library is one of the most genius js lib. It simulate file system with indexDB, but hide all the complicate DB operations. It will help you solve a lot of data issues... you know, like w/r offline files from browser...
SUPER GOOD!!!!!!!!!!!!!!!!!!!
